webapp/src/actions.js

```diff
@@ -29,6 +29,13 @@ export function setShowRHSAction(showRHSPluginAction) {
     };
 }
 
+export function setVisible(payload) {
```

Comment:
I think it makes sense to have a more specific function name here
```suggestion
export function setRhsVisible(payload) {
```